experiments/ClimaEarth/setup_run.jl

```diff
@@ -326,6 +326,7 @@ function setup_and_run(config_dict::AbstractDict)
             energy_check = energy_check,
             surface_elevation,
             use_land_diagnostics,
+            toml_files,
```

Comment:
It seems a little weird to me that the `toml_files` we pass to the bucket come from `get_atmos_args`. Can we either specify individual tomls for atmos and for land, or if we want them to be the same for both can we specify the tomls at the coupler level and pass them to both atmos and land?

Reply:
I agree, this is pretty hacky. Ideally we'd have one list of files that the coupler takes in and passes to both atmos and land but I wasn't sure how to accomplish that cleanly.

Reply:
We have the input `coupler_toml` that lets us specify a list of tomls at the coupler level. We could pass in multiple files there an pass them to atmos, land, etc. Maybe it could be a dictionary if we want to use the keys to determine which component the value goes to https://github.com/CliMA/ClimaCoupler.jl/blob/710a9f5c703ae40d7caea88b10cbc9339a630b3f/experiments/ClimaEarth/cli_options.jl#L23-L26

Reply:
Thanks, I think  the `coupler_toml` will be perfect but we will need to change the behavior to not prepend the `pkgdir(ClimaCoupler)` path to the toml paths.